connectivity/libraries/ppp/source/ppp_service.cpp

```diff
@@ -358,7 +358,7 @@ nsapi_error_t ppp_service::ppp_if_disconnect()
         ret = ppp_close(ppp_service_pcb, 0);
         if (ret == ERR_OK) {
             /* close call made, now let's catch the response in the status callback */
-            ppp_service_close_sem.try_acquire_for(PPP_TERMINATION_TIMEOUT);
+            ppp_service_close_sem.try_acquire_for(std::chrono::milliseconds(PPP_TERMINATION_TIMEOUT));
```

Comment:
I think it might be better to change the definition of `PPP_TERMINATION_TIMEOUT` to be of chrono type and then change the code that references it.

Reply:
I committed the changes you suggested